tests/misc_testsuite/gc/issue-10464.wast

```diff
@@ -1,14 +0,0 @@
-;;! gc = true
-
-(module
-  (type $a (struct))
-  (type $b (array structref))
-  (func (export "f")
-    struct.new_default $a
-    i32.const 536870911
-    array.new $b
-    drop
-  )
-)
-
-(assert_trap (invoke "f") "allocation size too large")
```

Comment:
Is it worth adapting this test to the new limit of 32-bits?

Reply:
`tests/misc_testsuite/gc/array-alloc-too-large.wast` already tests the 32-bit limit, so I think we are good.